Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/isBanned.md

$isBanned[user ID]
```

## Parameters 

> - `user ID ` `(Type : Snowflake || Flag : Required)` :the ID to check if banned or no.

Reviewer: ```suggestion
- `user ID ` `(Type: Snowflake || Flag: Required)`: The user to check its ban status.
```
